manifests/d/DeepL/DeepL/3.4.15088/DeepL.DeepL.locale.en-US.yaml

```
@@ -0,0 +1,25 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: DeepL.DeepL
+PackageVersion: 3.4.15088
+PackageLocale: en-US
+Publisher: DeepL GmbH

```

```suggestion
Publisher: DeepL SE
```
It also appears the Publisher got changed lately